dart/lib/src/sentry_client_attachment_processor.dart

```diff
@@ -0,0 +1,11 @@
+import 'dart:async';
+
+import './sentry_attachment/sentry_attachment.dart';
+import './protocol/sentry_event.dart';
+
+class SentryClientAttachmentProcessor {
```

Comment:
Is that the approach similar to iOS?

Reply:
Yeah that's where i have it from: https://github.com/getsentry/sentry-cocoa/blob/e2f115049093d6447abe798d1cfa76f40e5eb6d0/Sources/Sentry/include/SentryClient%2BPrivate.h#L11